cirq/optimizers/three_qubit_decomposition_test.py

```diff
@@ -54,6 +68,7 @@ def test_three_qubit_matrix_to_operations(u):
     assert num_two_qubit_gates <= 20, f"expected at most 20 CZ/CNOTs got {num_two_qubit_gates}"
 
 
+@_skip_if_scipy(has_cossin=False)
```

Comment:
Optional: The code in the except block in the 3-qubit decomposition has test coverage check disabled, so skipping the test on l.81 below should not cause coverage errors. However, if we skip this test, then I think coverage error will be reported (since we don't disable coverage on the code using cossin). This is not a problem as long as CI has scipy >= 1.5.0, which it should.

However, I'd add a comment here in case this turns out not to be the case and some poor soul ends up debugging what is going on.

Reply:
Thanks! I added a comment!